omnibus/files/private-chef-ctl-commands/version.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright:: Copyright (c) 2018 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_command_under_category "version", "general", "Display current version of Chef Server.", 2 do
+
+  # detect if running as a habitat service
+  if File.exist?('/hab/svc/chef-server-ctl/PID')
+    #code for populating version through /version endpoint
+    puts "running in hab"
+    ident_file = File.read('../IDENT')
+    version = "chef-server #{ident_file.split('/')[2]}"
+  else
+    version = File.readlines('/opt/opscode/version-manifest.txt')[0]
+  end
+
+  puts version
+
```

Comment:
nit: perhaps put an error handler after line 29 to better handle any `Errno::ENOENT`?

```
rescue Errno::ENOENT => ex
  puts "Error determining version! #{ex.message}"
```

The `rescue` would line up with the `add_command_under_category` and `end`

Reply:
adding error checking, I'm going to use e as the var and log instead of puts, to match how errors are handled in other parts of chef-server-ctl. For example rotate_credentials.rb